rulesdoc.md

```diff
@@ -2910,7 +2910,7 @@ You may choose to spend a minute casting this spell instead. If you do so the se
 - [Bramble](#Bramble)
 - [Breath](#Breath)
 - [Chilling Fog](#Chilling-Fog)
-- [Close Wounds](#Close-Wounds)
+- [Close Wound](#Close-Wound)
```

Comment:
This one has been bothering me. Is it Close Wound, or Close Wound**s**?

Reply:
I think it should be Wounds as it closes multiple.. sometimes